mapiproxy/servers/default/nspi/emsabp.c

```diff
@@ -863,7 +862,9 @@ _PUBLIC_ enum MAPISTATUS emsabp_table_fetch_attrs(TALLOC_CTX *mem_ctx, struct em
 				lpProps.value.l = 0x0;
 				break;
 			case PidTagDisplayName:
-				lpProps.value.lpszW = NULL;
+				/* This value is temporal to workaround PropertyRow_addprop internals */
+				/* It will be set to NULL after the call to PropertyRow_addprop */
+				lpProps.value.lpszW = "t";
```

Comment:
ok the "t" is indeed removed and I agree on the temporary aspect of this to be fixed after PR